bridges/ElloBridge.php

```diff
@@ -116,7 +116,7 @@ private function getAPIKey()
         $cacheFactory = new CacheFactory();
 
         $cache = $cacheFactory->create();
-        $cache->setScope(get_called_class());
```

Comment:
As in https://github.com/RSS-Bridge/rss-bridge/pull/2897, I would prefer to use fully-qualified class names internally (e.g. [`static::class`](https://www.php.net/manual/en/function.get-called-class.php#115790)) and convert any human-entered/config values to that single canonical representation as soon as possible in the code flow. The only downside I see is that caches would be invalidated once user updates to a RSS-Bridge with namespaces but that would be a one time thing.

Reply:
This particular scope is used by `FileCache` to build a file path in the cache folder e.g. `cache/ElloBridge/c120fc74ea1c80b5e1036dd1463c3f46.cache`

Reply:
I do no see an issue with that, `FileCache` should ensure that the scope is valid (e.g. replace slashes and whatever other characters file systems do not like with underscores).